configuration/pih/reports/sql/covid19encounterexport.sql

```diff
@@ -6,98 +6,1328 @@
+
+UPDATE temp_encounter
+SET 
+    neck_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '165983');
+
+-- chest
+UPDATE temp_encounter 
+SET 
+    chest = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '1123', 'fr');
+
+UPDATE temp_encounter
+SET 
+    chest_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '160689');
+
+-- cardiac
+UPDATE temp_encounter 
+SET 
+    cardiac = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '1124', 'fr');
+
+UPDATE temp_encounter
+SET 
+    cardiac_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '163046');
+
+-- abdominal
+UPDATE temp_encounter 
+SET 
+    abdominal = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '1125', 'fr');
+
+UPDATE temp_encounter
+SET 
+    abdominal_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '160947');
+
+-- urogenital
+UPDATE temp_encounter 
+SET 
+    urogenital = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '1126', 'fr');
+
+UPDATE temp_encounter
+SET 
+    urogenital_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '163047');
+
+-- rectal
+UPDATE temp_encounter 
+SET 
+    rectal = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '163746', 'fr');
+UPDATE temp_encounter 
+SET 
+    rectal_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '160961');
+
+-- musculoskeletal
+UPDATE temp_encounter 
+SET 
+    musculoskeletal = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '1128', 'fr');
+
+UPDATE temp_encounter
+SET 
+    musculoskeletal_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '163048');
+
+-- lymph
+UPDATE temp_encounter 
+SET 
+    lymph = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '1121', 'fr');
+
+UPDATE temp_encounter
+SET 
+    lymph_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '166005');
+
+-- skin
+UPDATE temp_encounter 
+SET 
+    skin = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '1120', 'fr');
+
+UPDATE temp_encounter
+SET 
+    skin_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '160981');
+
+-- neuro
+UPDATE temp_encounter 
+SET 
+    neuro = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '1129', 'fr');
+
+UPDATE temp_encounter
+SET 
+    neuro_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '163109');
+
+-- avpu
+UPDATE temp_encounter 
+SET 
+    avpu = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '162643', 'fr');
+
+-- other
+UPDATE temp_encounter 
+SET 
+    other_findings = OBS_VALUE_TEXT(encounter_id, 'CIEL', '163042');
+
+UPDATE temp_encounter 
+SET 
+    medications = OBS_VALUE_CODED_LIST(encounter_id, 'PIH', '1282', 'fr');
+
+UPDATE temp_encounter
+SET 
+    medication_comments = OBS_VALUE_TEXT(encounter_id,
+            'PIH',
+            'Medication comments (text)');
+
+-- supportive care
+UPDATE temp_encounter 
+SET 
+    supportive_care = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '165995', 'fr');
+
+-- o2therapy value
+UPDATE temp_encounter 
+SET 
+    o2therapy = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '165986');
+
+-- analgesic comments/description
+UPDATE temp_encounter 
+SET 
+    analgesic_specified = OBS_VALUE_TEXT(encounter_id, 'CIEL', '163206');
+
+##### IVF
```

Comment:
Hard for me to totally understand what is going on below in this IVF section with these temporary tables.  Might need more discussion.